pkg/kv/kvserver/metrics.go

```diff
@@ -1418,6 +1418,12 @@ cache will already have moved on to newer entries.
 		Measurement: "Bytes",
 		Unit:        metric.Unit_BYTES,
 	}
+	metaRaftStorageError = metric.Metadata{
+		Name:        "raft.storage.error",
+		Help:        "Number of calls to the raft.Storage API that returned an error",
```

Comment:
nit: the "raft.Storage API" part seems a bit too low-level for users, consider perhaps "Number of Raft storage errors".